charts/nublado/templates/jupyterhub/deployment.template.yml

```diff
@@ -219,13 +226,11 @@ spec:
           persistentVolumeClaim:
            claimName: {{ template "nublado.fullname" . }}-hub-physpvc
 {{ end }}
-        - name: fs-mounts
-          configMap:
-            defaultMode: 0444
-            items:
-            - key: mountpoints.json
-              path: mountpoints.json
-            name: {{ template "nublado.fullname" . }}-fs-mounts
```

Comment:
Why are you deleting this? It seems unrelated.

Reply:
That was a mistake.  Restored.